app/assets/javascripts/discourse/app/components/header/topic/info.gjs

```diff
@@ -139,25 +139,27 @@ export default class Info extends Component {
               )
             }}
               <div class="categories-wrapper">
-                {{#if @topicInfo.category.parentCategory}}
-                  {{#if
-                    (and
-                      @topicInfo.category.parentCategory.parentCategory
-                      this.site.desktopView
-                    )
-                  }}
+                <PluginOutlet @name="header-categories-wrapper">
```

Comment:
Do we need some outletArgs here? Like `@outletArgs={{hash category=@topicInfo.category}}` ?